src/network/networkctl.c

```diff
@@ -1706,6 +1706,7 @@ static int link_status_one(
         (void) sd_network_link_get_route_domains(info->ifindex, &route_domains);
         (void) sd_network_link_get_ntp(info->ifindex, &ntp);
         (void) sd_network_link_get_sip(info->ifindex, &sip);
+        (void) sd_network_link_get_captive_portal(info->ifindex, &captive_portal);
```

Comment:
Not used??

Reply:
Added a "Captive Portal" field to the `networkctl status <link>` output, so now this is used.

Reply:
```
$ networkctl status mv-enp4s0
[...]
     Activation Policy: up
   Required For Online: yes
        Captive Portal: http://example.com
       DHCP4 Client ID: IAID:0xc3784450/DUID
     DHCP6 Client DUID: DUID-EN/Vendor:0000ab11f29abad7f3b60b5d0000
```